include/odp/api/spec/classification.h

```diff
@@ -317,11 +315,37 @@ typedef enum {
 	ODP_COS_FHDR_USER	/**< Application-specific header field(s) */
 } odp_cos_hdr_flow_fields_t;
 
+/**
+ * Enumeration of actions for CoS.
+ */
+typedef enum {
+	/**
+	 * Accept packet
+	 *
+	 * Packets that arrive in the CoS are accepted and their processing
+	 * continues towards other chained CoSes or the destination queue.
+	 */
+	ODP_COS_ACTION_ACCEPT,
```

Comment:
Another naming option for accept CoS could be ACTION_ENQUEUE. It would allow us to add other accept type actions later.

In general, action description would need to say in someway that: the last "matching" CoS defines the action for a packet. I.e. packet is dropped only when the CoS of the last matching rule has drop action (there may be other CoSes with drop action on the way).

Reply:
ACTION_ENQUEUE in v5.